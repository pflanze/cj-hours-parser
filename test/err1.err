Error: can't process file "test/err1.txt"

Caused by:
    line 14: garbage after closing delimiters: " garble"
