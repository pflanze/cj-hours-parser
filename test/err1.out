item=SingleLineComment(" this is a comment")
    =SingleLineComment(" this is a comment")
item=MultiLineComment(["this is a multiline", "  comment"])
    =MultiLineComment(["this is a multiline", "  comment"])
item=Other([Minus, Time(NT(12:44:00)), Rest("test  some time")])
    =Other([Subitem(L(0, 6, 0), Minus), Subitem(L(0, 6, 1), Time(NT(12:44:00))), Subitem(L(0, 6, 7), Rest("test  some time"))])
item=Empty(1)
    =Empty(1)
item=MultiLineComment(["some more commenting"])
    =MultiLineComment(["some more commenting"])
item=SingleLineComment("[and yet more]")
    =SingleLineComment("[and yet more]")
